### ISSUE TITLE
modernize: add enable linter setting

### ISSUE THREAD

**Issue by alexandear** [golangci/golangci-lint#6379]
### Welcome

- [x] Yes, I've searched similar [issues on GitHub](https://github.com/golangci/golangci-lint/issues) and didn't find any.
- [x] I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)

### How did you install golangci-lint?

Other

### Your feature request related to a problem? Please describe

Currently, it's impossible to enable these `modernize` analyzers that are present in `golang.org/x/tools@v0.42.0`, but missing from [modernize.Suite](https://cs.opensource.google/go/x/tools/+/refs/tags/v0.42.0:go/analysis/passes/modernize/modernize.go;l=34):

- [appendclipped](https://pkg.go.dev/golang.org/x/tools@v0.42.0/go/analysis/passes/modernize#hdr-Analyzer_appendclipped)
- [bloop](https://pkg.go.dev/golang.org/x/tools@v0.42.0/go/analysis/passes/modernize#hdr-Analyzer_bloop)
- [slicesdelete](https://pkg.go.dev/golang.org/x/tools@v0.42.0/go/analysis/passes/modernize#hdr-Analyzer_slicesdelete)

### Describe the solution you'd like

Add the `linters.settings.modernize.enable` config option, similar to [`linters.settings.govet.enable`](https://github.com/golangci/golangci-lint/blob/57daeeeb122e8c7602e57fcf9ebd77de9ee53783/.golangci.reference.yml#L1726). This would allow users to enable missing analyzers explicitly:

```yaml
linters:
  settings:
    modernize:
      # These analyzers are enabled in addition to the default analyzers from the modernize.Suite list.
      enable:
        - appendclipped
        - bloop
        - slicesdelete
```

### Describe alternatives you've considered

Run `appendclipped`, `bloop`, `slicesdelete` via `go vet tool`.

### Supporter

- [ ] I am a [sponsor/backer](https://donate.golangci.org) of this project.

**Comment by ldez** [comment]
We only support the analyzers from the `modernize.Suite`, this is expected because other produces unexpected changes as documented inside the suite code.
